tests/console/sshd.pm

```diff
@@ -8,57 +8,78 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Very old test to verify sshd starts up.
+# Summary: Test to verify sshd starts and accepts connections.
 #  We need this test to succeed for followup tests using ssh localhost
-# Maintainer: Stephan Kulow <coolo@suse.de>
+#  This regression test has also an interactive part (in VirtIO console)
+# Maintainer: Stephan Kulow <coolo@suse.de>, Pavel Dostál <pdostal@suse.cz>
 
+use 5.018;
```

Comment:
any reason for this specific version ?

Reply:
Good morning, no, it was part of the official documentation but I just found out it's not necessary so I removed it.